aldryn_newsblog/admin.py

```diff
@@ -1,36 +1,42 @@
 from django.contrib import admin
 from django.utils.translation import ugettext_lazy as _
-
 from cms.admin.placeholderadmin import FrontendEditableAdminMixin
-from parler.admin import TranslatableAdmin
 
+from parler.admin import TranslatableAdmin
 from aldryn_apphooks_config.admin import BaseAppHookConfig
 from aldryn_people.models import Person
 from aldryn_reversion.admin import VersionedPlaceholderAdminMixin
-
 from . import models
 
 
 def make_published(modeladmin, request, queryset):
     queryset.update(is_published=True)
+
+
```

Comment:
These follow immediately after the function definition for a reason - it's a common idiom in Python to do that when annotating an object.

Also, in general, it's best not to mix formatting changes with your substantial PRs, since they only make it harder to review.